flink-dist/pom.xml

```diff
@@ -137,8 +137,8 @@ under the License.
 			<version>${project.version}</version>
 			<exclusions>
 				<exclusion>
-					<groupId>org.apache.flink</groupId>
-					<artifactId>flink-shaded-hadoop-2</artifactId>
+					<groupId>org.apache.hadoop</groupId>
+					<artifactId>*</artifactId>
```

Comment:
This breaks compatibility with maven 3.1

Reply:
Oh, I didn't know that this is a newer feature. However, we have already a case in master with a star exclude: https://github.com/apache/flink/blob/master/flink-connectors/flink-connector-elasticsearch-base/pom.xml#L68

Can we raise the minimum maven version to [3.2.1](https://maven.apache.org/docs/3.2.1/release-notes.html) then?

Reply:
hmm, should've read MNG-3832 more carefully; it _does_ work with all maven 3 versions, just prints a warning on everything below 3.2.1 .